src/event_handler.cpp

```diff
@@ -182,15 +164,17 @@ void	EventHandler::HandlePollHupEvent(pollfd entry) {
 	}
 }
 
-void	EventHandler::Detach(pollfd event) {
-	std::cout << "connection hang up " << event.fd << std::endl;
+void	EventHandler::Detach(pollfd entry) {
+	std::cout << "connection hang up " << entry.fd << std::endl;
 	int target_index = 0;
 	for (int i = 0; i < (int)poll_fd_.size(); i++)
 	{
```

Comment:
万が一ですが、見つからなかった場合の対処もすると良いでしょう。
(一気に提案できなかったので分割しています)
その1
```suggestion
	std::vector<struct pollfd>::size_type index = 0;
	for (; index < poll_fd_.size(); index++) {
```